native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -41,7 +43,6 @@ const {
 const styles = StyleSheet.create({
   card: {
     // TODO: update once tokens are available
-    borderRadius: 8,
```

Comment:
No more borders?

Reply:
it has been moved to the common styles because both the TouchableHighlight and View are using it